Formula/teaxyz.rb

```diff
@@ -0,0 +1,50 @@
+class Teaxyz < Formula
+  desc "Unified package manager"
+  homepage "https://tea.xyz"
+  url "https://github.com/teaxyz/cli/releases/download/v0.13.6/tea-0.13.6.tar.xz"
+  sha256 "88ee0e93a49e2bea80cf75d971bf8fc236c024a42727dafbaf5f7dee4f503d8e"
+  license "Apache-2.0"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "deno" => :build
+
+  conflicts_with "tea", because: "both install `tea` binaries"
+
+  def install
+    system "deno", "compile",
+      "--allow-read",
+      "--allow-write",
+      "--allow-net",
+      "--allow-run",
+      "--allow-env",
+      "--unstable",
+      "--import-map=import-map.json",
+      "--output", bin/"tea",
+      "src/app.ts"
+  end
+
+  def post_install
+    system "#{bin}/tea", "--sync"
+  end
+
+  def caveats
```

Comment:
This should be in the upstream docs, it's not brew specific.

Reply:
I'll add it to the caveats instead.

Reply:
I was actually talking about the caveats, which aren't really caveats.

Reply:
> To use tea’s virtual environment manager functionality, you need to add one
> line to your shell rc:
>
>.   https://github.com/teaxyz/cli#getting-started

Is a caveat in that with our own installer this is done for you. Do you still wish that this is not broadcast?

Reply:
k removed them. We'll just emphasize that this is needed in the docs as you suggest.